### PR TITLE
Add spacing and background to people.

### PR DESCRIPTION
**Pull request by simonsasse**
Adding 10px between all peoples.
adding grey background when unfolded.
removing blue border that appeared when clicked on a person.

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -349,7 +350,14 @@ main {
...
 .person-bio {
-    margin-left: $spacing-unit;
+  margin-left: $spacing-unit;
+  margin-right: $spacing-unit;
+}
+
+details.person[open]{
+background-color: #eaeaea;
+border-radius: 6px;
+margin-bottom:10px;
```
Comment (marehr) [inline assets/css/style.scss:360]:
```suggestion
  background-color: #eaeaea;
  border-radius: 6px;
  margin-bottom:10px;
```

File: assets/css/style.scss [hunk 2]
```diff
@@ -333,7 +333,8 @@ main {
     padding: 0.5em;
     @include relative-font-size(1.5);
     border-radius: 6px;
-
+    margin-bottom:10px;
+    outline:none;
```
Comment (marehr) [inline assets/css/style.scss:337]:
```suggestion
    margin-bottom: 10px;
    outline: none;
```

File: assets/css/style.scss [hunk 3]
```diff
@@ -333,7 +333,8 @@ main {
     padding: 0.5em;
     @include relative-font-size(1.5);
     border-radius: 6px;
-
+    margin-bottom: 10px;
```
Comment (marehr) [inline assets/css/style.scss:336]:
```suggestion
    margin-bottom: $spacing-unit;
```

File: assets/css/style.scss [hunk 4]
```diff
@@ -333,7 +333,8 @@ main {
     padding: 0.5em;
     @include relative-font-size(1.5);
     border-radius: 6px;
-
+    margin-bottom: $spacing-unit;
```
Comment (marehr) [inline assets/css/style.scss:336]:
```suggestion
```

File: assets/css/style.scss [hunk 5]
```diff
@@ -349,7 +349,14 @@ main {
...
 .person-bio {
-    margin-left: $spacing-unit;
+  margin-left: $spacing-unit;
+  margin-right: $spacing-unit;
+}
+
+details.person[open]{
+  background-color: #eaeaea;
+  border-radius: 6px;
+  margin-bottom: $spacing-unit;
```
Comment (marehr) [inline assets/css/style.scss:359]:
```suggestion
    margin-left: $spacing-unit;
    margin-right: $spacing-unit;
}

details.person[open]{
    background-color: #eaeaea;
    border-radius: 6px;
    margin-bottom: $spacing-unit;
```

**Comment by marehr** [comment]
We currently don't have the concept of filled out "boxes" (the old website also hadn't any filled out boxes) and I'm not sure how we can keep a "sleek" design.

I'm not sure whether 

grafik

grafik

grafik

grafik

might look better.

on the old site we had:

grafik

**Comment by simonsasse** [comment]
So when using the yellow border, the content itself jumped, because the border forces it to rescale a bit. I think the separator line is quite nice? Sorry for the indents !

File: assets/css/style.scss [hunk 6]
```diff
@@ -333,7 +333,8 @@ main {
     padding: 0.5em;
     @include relative-font-size(1.5);
     border-radius: 6px;
-
+    margin-bottom: $spacing-unit;
+    outline: none;
     &:hover {
```
Comment (marehr) [inline assets/css/style.scss:338]:
```suggestion
    // highlight the summary of a person when hovering, or if the person is unfolded
    &:hover, details.person[open] & {
```

File: assets/css/style.scss [hunk 7]
```diff
@@ -350,6 +351,19 @@ main {
...
+details.person[open]{
+    border-style: solid;
+    border-width: 0px;
+    border-bottom-width: 1px;
+    border-color: $seqan-color;
+    margin-bottom: $spacing-unit;
+    // locks the background color of the summary to yellow, if person is unfolded.
+    .person-summary {
+        background-color: $seqan-color;
+    }
```
Comment (marehr) [inline assets/css/style.scss:366]:
```suggestion
```

File: assets/css/style.scss [hunk 8]
```diff
@@ -350,6 +352,15 @@ main {
...
 .person-bio {
     margin-left: $spacing-unit;
+    margin-right: $spacing-unit;
+}
+
+details.person[open]{
+    border-style: solid;
+    border-width: 0px;
+    border-bottom-width: 1px;
+    border-color: $seqan-color;
```
Comment (marehr) [inline assets/css/style.scss:362]:
```suggestion
    border-bottom: 1px solid $seqan-color;
```

File: assets/css/style.scss [hunk 9]
```diff
@@ -333,8 +333,10 @@ main {
     padding: 0.5em;
     @include relative-font-size(1.5);
     border-radius: 6px;
-
-    &:hover {
+    margin-bottom: $spacing-unit;
+    outline: none;
+    // highlight the summary of a person when hovering, or if the person is unfolded
```
Comment (marehr) [inline assets/css/style.scss:338]:
```suggestion

    // highlight the summary of a person when hovering, or if the person is unfolded
```

File: assets/css/style.scss [hunk 10]
```diff
@@ -350,6 +353,12 @@ main {
 
 .person-bio {
     margin-left: $spacing-unit;
+    margin-right: $spacing-unit;
+}
+
+details.person[open]{
```
Comment (marehr) [inline assets/css/style.scss:359]:
```suggestion
details.person[open] {
```

**Review by marehr — APPROVED** [review]
Thank you :)))

**Pull request merged**